lib/importmap/map.rb

```diff
@@ -41,8 +41,60 @@ def pin_all_from(dir, under: nil, to: nil, preload: true)
   # resolve for different asset hosts, you can pass in a custom `cache_key` to vary the cache used by this method for
   # the different cases.
   def preloaded_module_paths(resolver:, entry_point: "application", cache_key: :preloaded_module_paths)
```

Comment:
This method can probably be deprecated now since the library isn't using anymore